### ISSUE TITLE
Is it possible to add a stylistic set for the double ampersands && when using code ligatures?

### ISSUE THREAD

**Issue by enriqueMartinezInter** [arrowtype/recursive#437]
I love this font so much but I was wondering if a stylistic set can be added when using code ligatures for the double ampersands whether it would be applied or not to those characters  when they represent the AND operator.

**Comment by arrowtype** [comment]
Thanks for the kind words!

So, are you thinking of an option to turn *off* the && ligature when other code ligatures are active? Or, are you looking for something that would turn off the && ligature only when it is used for the `and` operator?

**Comment by enriqueMartinezInter** [comment]
Yes, exactly, an option to turn off the && ligature when other code ligatures are active, so for instance if using dlig and then adding say ss15 would turn that specific ligature off.

**Comment by arrowtype** [comment]
Okay, thanks for clarifying! You’re not the first to ask about it, so I’ll probably try to do this at some point.

Just to fully understand, is your preference because you find the shape of the ligature distracting, or that it is specifically a problem for it to be a ligature? (For example, does a single glyph disrupt syntax theme colors, or come up in other strings, or something else?)

**Comment by enriqueMartinezInter** [comment]
Yes, I find it easier to read when both characters are side by side and not as a code ligature, all other code ligatures are great! This is the only one I wish it could be at least optional. Maybe removing it from the ligatures set would be too much to ask :D, but optional would be great.

**Comment by arrowtype** [comment]
Sounds good! I’ll look into it when I have the chance. 👍

**Comment by Gahotx** [comment]
> So, are you thinking of an option to turn _off_ the && ligature when other code ligatures are active? Or, are you looking for something that would turn off the && ligature only when it is used for the `and` operator?

Hello, this font is very nice and I also need this feature. Will this feature be updated?
